manifests/cirunner.pp

```diff
@@ -82,12 +74,9 @@
         include apt
         ensure_packages('apt-transport-https')
 
-        $distid = downcase($::lsbdistid)
-
-        ::apt::source { 'apt_gitlabci':
+        apt::source { 'apt_gitlabci':
           comment  => 'GitlabCI Runner Repo',
-          location => "${repo_base_url}/runner/${package_name}/${distid}/",
-          release  => $::lsbdistcodename,
+          location => "${repo_base_url}/runner/${package_name}/${::lsbdistid.downcase}/",
```

Comment:
This is a legacy fact. Can you switch it to a modern fact please?

Reply:
If you want to fix the sped tests, sure. I tried to used the modern facts, but the lib rspec-puppet-facts doesn‘t provide modern facts and I get tons of error in travis.

Reply:
If you want to fix the sped tests, sure. I tried to used the modern facts, but the lib rspec-puppet-facts doesn‘t provide modern facts and I get tons of error in travis.

Reply:
nah, sounds like out of scope for this PR in that case. We'll fix it in another issue/PR.